xlml/apis/test_config.py

```diff
@@ -480,10 +476,45 @@ class JSonnetGpuTest(TestConfig[Gpu]):
   """
 
   test_name: str
-  test_command: List[str]
   entrypoint_script: List[str]
+  test_command: List[str]
   docker_image: str
+  accelerator: Gpu
+  time_out_in_min: str
   num_hosts: int = 1
+  runtime_version: str = 'n/a'
```

Comment:
`runtime_version`, `network`, and `subnetwork` should not be necessary for GKE. That's all specified in terraform and baked into the cluster configuration.